system-int-test/test/cmr/system_int_test/search/scrolling_search_test.clj

```diff
@@ -135,6 +135,46 @@
             (is (= (count all-grans) hits))
             (is (data2-core/refs-match? [gran3 gran4] result))))
 
+        (testing "Remaining results returned on last search"
```

Comment:
It's not obvious to me how retrieving the next page of results is going to work in every format based on what's cached. I think we should sanity check that deferred scrolling returns the correct results for all of the granule formats (xml references - done, echo10, json, and umm-json).

Reply:
I tested it with echo10, iso, json, native for granules and they all worked. The first page will be in whatever format you originally requested, so making a deferred request in one format then asking for the first page in another format is not supported - though I'm not sure why you would want to do that.